src/api/spec/models/workflow/step/branch_package_step_spec.rb

```diff
@@ -268,7 +268,8 @@
               { 'action' => 'opened', 'commit_sha' => '456', 'event' => 'pull_request', 'pr_number' => 1, 'scm' => 'github', 'source_repository_full_name' => 'reponame' }
             end
             let(:update_payload) do
-              { 'action' => 'synchronize', 'commit_sha' => '456', 'event' => 'pull_request', 'pr_number' => 1, 'scm' => 'github', 'source_repository_full_name' => 'reponame' }
+              { 'action' => 'synchronize', 'commit_sha' => '456', 'event' => 'pull_request', 'pr_number' => 1, 'scm' => 'github', 'source_repository_full_name' => 'reponame',
+                'workflow_filters' => [] }
```

Comment:
It doesn't change the results of the specs, but `workflow_filters` should contain a hash, not an array.
```suggestion
                'workflow_filters' => {} }
```

Reply:
Done